File: Napakalaki-Ruby/lib/prize.rb
# Javier Aranda Izquierdo
module NapakalakiGame

class Prize
  
  def initialize(treasures, levels)
     @treasures=treasures
     @levels=levels
  end
  
  #Consultores
  attr_reader:treasures # Metodo getTreasures
  attr_reader:level # Metodo getLevels
  
  def to_s
  "Tesoros ganados: #{@treasures} \nNiveles ganados: #{@level}"
  end
    

end

end

File: Napakalaki-Java/src/napakalaki/Prize.java
package napakalaki;

/**
 *
 * @author Javier Aranda
 */
public class Prize {
    // Atributos
    private int treasures; //
    private int level;
    
    //Constructor
    public Prize(int treasures, int level){
        this.treasures=treasures;
        this.level=level;
    };
    
    public int getTreasures(){
        return treasures;
    }
    
    public int getLevel(){
        return level;
    }
    
}
